aspnetcore/security/app-secrets.md

```diff
@@ -201,21 +202,13 @@ The `Movies:ConnectionString` and `Movies:ServiceApiKey` secrets are mapped to t
 
 ## String replacement with secrets
 
-Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` may include a password for the specified user:
-
-[!code-json[](~/security/app-secrets/samples/3.x/UserSecrets/appsettings-unsecure.json?highlight=3)]
-
-A more secure approach is to store the password as a secret. For example:
+Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` should not include a password. Instead, store the password as a secret, and include the password in the connection string at runtime. For example:
 
 ```dotnetcli
-dotnet user-secrets set "DbPassword" "pass123"
+dotnet user-secrets set "DbPassword" "`<secret value>`"
```

Comment:
For bonus points, replace "`<secret value>`"
with
`"\x00``<secret value>``\"`  which should throw an error, although maybe `<secret value>` will throw. We don't want them copy/paste something that works.

Reply:
If someone is determined to use it as an actual password, I'm afraid we would only be giving them a particularly strong one.